flink-runtime/src/main/java/org/apache/flink/runtime/checkpoint/metadata/MetadataV2V3SerializerBase.java

```diff
@@ -117,6 +117,8 @@ public abstract class MetadataV2V3SerializerBase {
     private static final byte INCREMENTAL_KEY_GROUPS_HANDLE_V2 = 11;
     // KEY_GROUPS_HANDLE_V2 is introduced to add new field of stateHandleId.
     private static final byte KEY_GROUPS_HANDLE_V2 = 12;
+    private static final byte CHANGELOG_HANDLE_V2 = 13;
+    private static final byte CHANGELOG_FILE_INCREMENT_HANDLE_V2 = 14;
```

Comment:
Can we add descriptions in the code why we need to introduce these two new flags here?